src/core/agent/services/credentialService.ts

```diff
@@ -139,6 +141,25 @@ class CredentialService extends AgentService {
     return metadata;
   }
 
+  @OnlineOnly
+  async getKeriCredentialNotifications(
```

Comment:
I renamed this to `getUnhandledIpexGrantNotifications` so this is duplicated now - we can remove this function and the tests